spswarehouse/powerschool/calpads.py

```diff
@@ -0,0 +1,220 @@
+                                                                    )
+    elif(report_name == "Student Program Records"):
+        return download_calpads_report_for_school_student_program_records(
+                                                                    driver=driver,
+                                                                    report_name=report_name,
+                                                                    file_postfix=file_postfix, 
+                                                                    destination_directory_path=destination_directory_path, 
+                                                                    report_parameters=report_parameters,
+                                                                    validation_only_run=validation_only_run
+                                                                    )
+    elif(report_name == "Course Section Records"):
+        return download_calpads_report_for_school_course_section_records(
+                                                                    driver=driver,
+                                                                    report_name=report_name,
+                                                                    file_postfix=file_postfix, 
+                                                                    destination_directory_path=destination_directory_path, 
+                                                                    report_parameters=report_parameters,
+                                                                    validation_only_run=validation_only_run
+                                                                    )
+    elif(report_name == "Student Course Section Records"):
+        return download_calpads_report_for_school_student_course_section_records(
+                                                                    driver=driver,
+                                                                    report_name=report_name,
+                                                                    file_postfix=file_postfix, 
+                                                                    destination_directory_path=destination_directory_path, 
+                                                                    report_parameters=report_parameters,
+                                                                    school_subdistrict_name=school_subdistrict_name, # This is an additional parameter compared to the function calls earlier in the if-else tree
+                                                                    validation_only_run=validation_only_run,
+                                                                    )            
+    else:
+        raise Exception("CALPADS report name not supported")
+
+def download_calpads_report_for_school_student_incident_records_sinc(driver: WebDriver, file_postfix: str, destination_directory_path: str, 
```

Comment:
I think all these report specific functions are internal?

Also, wondering if the names could be shorter. Since this is already a CALPADS file, we can probably reduce the amount of information in the function name?